Allow singular `lb` in `from_str()` (#53)
rust-embedded-community/rust-measurements: CHANGELOG.md

## Added

- Derive `Default` for all measurement structs

> - make `lb` an option for mass in FromStr implementation (works the same as `lbs` does currently)

Reviewer: ```suggestion
- Accept `lb` as a synonym for `lbs` for mass.
```
